apps/teams-test-app/src/components/privateApis/ExternalAppAuthenticationAPIs.tsx

```diff
@@ -65,6 +65,46 @@ const AuthenticateAndResendRequest = (): React.ReactElement =>
     }),
   });
 
+const AuthenticateWithOauth2 = (): React.ReactElement =>
+  ApiWithTextInput<{
+    titleId: string;
+    oauthConfigId: string;
+    oauthWindowParameters: {
+      width?: number;
+      height?: number;
```

Comment:
Does this also need to include `isExternal`?

Reply:
updated. Thank you!